{"uuid": "941daeeb-b6a2-4b2e-ac7f-cc3aaaa275ad", "children": ["22ddeb0c-aaf5-403c-af2e-2db96d0e8749"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 99\nCurrent browser version is 124.0.6367.118 with binary path /usr/bin/google-chrome\nStacktrace:\n#0 0x6367852dd7d3 <unknown>\n#1 0x636785039688 <unknown>\n#2 0x6367850606a0 <unknown>\n#3 0x63678505be36 <unknown>\n#4 0x63678505868f <unknown>\n#5 0x636785093355 <unknown>\n#6 0x63678508d5a3 <unknown>\n#7 0x636785062ddc <unknown>\n#8 0x636785063de5 <unknown>\n#9 0x63678530e49d <unknown>\n#10 0x63678532760c <unknown>\n#11 0x636785310205 <unknown>\n#12 0x636785327ee5 <unknown>\n#13 0x636785304070 <unknown>\n#14 0x636785343488 <unknown>\n#15 0x63678534360c <unknown>\n#16 0x63678535cc6d <unknown>\n#17 0x7081cb694ac3 <unknown>\n\n", "trace": "  File \"/home/atish/PycharmProjects/Py2xWebAutomationFramework/pythonProject/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/atish/PycharmProjects/Py2xWebAutomationFramework/pythonProject/.venv/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/atish/PycharmProjects/Py2xWebAutomationFramework/pythonProject/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/atish/PycharmProjects/Py2xWebAutomationFramework/pythonProject/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/atish/PycharmProjects/Py2xWebAutomationFramework/pythonProject/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 884, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/atish/PycharmProjects/Py2xWebAutomationFramework/pythonProject/tests/vwoLoginTests/test_vwo_login.py\", line 16, in setup\n    driver = webdriver.Chrome()\n  File \"/home/atish/PycharmProjects/Py2xWebAutomationFramework/pythonProject/.venv/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"/home/atish/PycharmProjects/Py2xWebAutomationFramework/pythonProject/.venv/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py\", line 66, in __init__\n    super().__init__(command_executor=executor, options=options)\n  File \"/home/atish/PycharmProjects/Py2xWebAutomationFramework/pythonProject/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 208, in __init__\n    self.start_session(capabilities)\n  File \"/home/atish/PycharmProjects/Py2xWebAutomationFramework/pythonProject/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 292, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n  File \"/home/atish/PycharmProjects/Py2xWebAutomationFramework/pythonProject/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/home/atish/PycharmProjects/Py2xWebAutomationFramework/pythonProject/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1715873556679, "stop": 1715873557720}], "start": 1715873556679, "stop": 1715873557778}